From cea52aa1922ab3bed301c719832ddaedb5402771 Mon Sep 17 00:00:00 2001
From: Nick Verse <nikoskalomiris@gmail.com>
Date: Wed, 31 Aug 2016 14:02:33 +0300
Subject: [PATCH] Fix clocks and voltages after OC (1)

* Add 1190400 frequency. Needed for cpufreq_limit on touchscreen press.
* Fix initial voltage for regulator
---
 .../dts/samsung/msm8916/msm8916-regulator.dtsi     | 33 +++++++++++-----------
 1 file changed, 17 insertions(+), 16 deletions(-)

diff --git a/arch/arm/boot/dts/samsung/msm8916/msm8916-regulator.dtsi b/arch/arm/boot/dts/samsung/msm8916/msm8916-regulator.dtsi
index b65d58c..42908f2 100755
--- a/arch/arm/boot/dts/samsung/msm8916/msm8916-regulator.dtsi
+++ b/arch/arm/boot/dts/samsung/msm8916/msm8916-regulator.dtsi
@@ -17,7 +17,7 @@
 			compatible = "qcom,spm-regulator";
 			regulator-name = "8916_s2";
 			reg = <0x1700 0x100>;
-			regulator-min-microvolt = <1000000>;
+			regulator-min-microvolt = <1050000>;
 			regulator-max-microvolt = <1385000>;
 		};
 	};
@@ -48,10 +48,10 @@
 		regulator-name = "apc_corner";
 		qcom,cpr-fuse-corners = <3>;
 		regulator-min-microvolt = <1>;
-		regulator-max-microvolt = <16>;
+		regulator-max-microvolt = <17>;
 
-		qcom,cpr-voltage-ceiling = <1000000 1150000 1385000>;
-		qcom,cpr-voltage-floor = <1000000 1050000 1275000>;
+		qcom,cpr-voltage-ceiling = <1050000 1150000 1385000>;
+		qcom,cpr-voltage-floor = <1050000 1150000 1385000>;
 		vdd-apc-supply = <&pm8916_s2>;
 
 		qcom,vdd-mx-corner-map = <4 5 7>;
@@ -83,9 +83,9 @@
 					<27 36 6 0>,
 					<27 18 6 0>,
 					<27 0 6 0>;
-		qcom,cpr-init-voltage-ref = <1000000 1150000 1385000>;
+		qcom,cpr-init-voltage-ref = <1050000 1150000 1385000>;
 		qcom,cpr-init-voltage-step = <10000>;
-		qcom,cpr-corner-map = <1 1 2 2 3 3 3 3 3 3 3 3 3 3 3 3>;
+		qcom,cpr-corner-map = <1 1 2 2 3 3 3 3 3 3 3 3 3 3 3 3 3>;
 		qcom,cpr-corner-frequency-map =
 					<1 200000000>,
 					<2 400000000>,
@@ -94,18 +94,19 @@
 					<5 998400000>,
 					<6 1094400000>,
 					<7 1152000000>,
-					<8 1209600000>,
-					<9 1248000000>,
-					<10 1363200000>,
-					<11 1401600000>,
-					<12 1497600000>,
-					<13 1593600000>,
-					<14 1689600000>,
-					<15 1785600000>,
-					<16 1881600000>;
+					<8 1190400000>,
+					<9 1209600000>,
+					<10 1248000000>,
+					<11 1363200000>,
+					<12 1401600000>,
+					<13 1497600000>,
+					<14 1593600000>,
+					<15 1689600000>,
+					<16 1785600000>,
+					<17 1881600000>;
 		qcom,speed-bin-fuse-sel = <1 34 3 0>;
 		qcom,cpr-speed-bin-max-corners =
-					<0 0 2 4 16>;
+					<0 0 2 4 17>;
 		qcom,cpr-quot-adjust-scaling-factor-max = <650>;
 		qcom,cpr-enable;
 	};
